clients/include/testing_rot_ex.hpp

```diff
@@ -75,16 +75,15 @@ hipblasStatus_t testing_rot_ex_template(const Arguments& arg)
     host_vector<Tcs> hc(1);
     host_vector<Tcs> hs(1);
 
-    srand(1);
-    hipblas_init<Tx>(hx_host, 1, N, abs_incx);
-    hipblas_init<Ty>(hy_host, 1, N, abs_incy);
-
     // Random alpha (0 - 10)
     host_vector<int> alpha(1);
-    hipblas_init<int>(alpha, 1, 1, 1);
 
-    hipblas_init<Tcs>(hc, 1, 1, 1);
-    hipblas_init<Tcs>(hs, 1, 1, 1);
+    hipblas_init_vector(hx_host, arg, N, abs_incx, 0, 1, hipblas_client_alpha_sets_nan, true);
+    hipblas_init_vector(hy_host, arg, N, abs_incy, 0, 1, hipblas_client_alpha_sets_nan, false);
+    hipblas_init_vector(alpha, arg, 1, 1, 0, 1, hipblas_client_alpha_sets_nan, false);
```

Comment:
I think alpha would use never_sets_nan, it is only for testing alpha=0 like tests that would non even reference x & y

Reply:
Maybe they should all be never as you don't use alpha as rotation angle anymore?

Reply:
Almost all Level 1 functions in rocBLAS had NaN initialization in the pre_checkin test and most of the functions neither have `alpha` nor `beta` parameters. Therefore, added  `hipblas_client_alpha_sets_nan` to initialize vectors with NaN's. I can also add another enum in `rocblas_check_nan_init` for Level 1 NaN initialization if this is misleading.

Reply:
I see now that rocBLAS skips the NaN checks in this case. I think it's best to remove `hipblas_client_alpha_sets_nan` for the rot* cases for now, even though NaN tests haven't been added to the test suite yet.